### PR TITLE
Blog Post: Announcing the OpenSats Education Initiative

### PR DESCRIPTION
**Pull request by dergigi**
To be reviewed by @OpenSats/board & sanity checked by @OpenSats/volunteers 

---

**Build preview:**
- [/blog/announcing-the-opensats-education-initiative](https://os-website-git-education-initiative-open-sats.vercel.app/blog/announcing-the-opensats-education-initiative)

**Social preview:**
38-education

### CODE REVIEW

**Review by islandbitcoin — APPROVED** [review]
Edits look good. ACK

**Comment by elaineo** [comment]
Looks good to me

**Comment by dergigi** [comment]
Ready for final sanity check by @OpenSats/volunteers

**Comment by bitstein** [comment]
The SNI website is APGL, but the content produced by us (not otherwise licensed) is CC-BY-SA 4.0, fwiw. Not sure if that is the more pertinent license to showcase.

**Review by Arvin21M — APPROVED** [review]
Outside of @bitstein's concern re: license, the article reads fine, links all work and point to the right place. Header image is dope. Very happy about these educational grants. 

Looks great to me! 🚀

**Comment by dergigi** [comment]
> The SNI website is APGL, but the content produced by us (not otherwise licensed) is CC-BY-SA 4.0

Fixed! Thanks @bitstein 🙏🧡

**Comment by btcschellingpt** [comment]
So pleased to see this - lot of people will benefit from the amplification of learning initiatives LFG!!

File: data/blog/announcing-the-opensats-education-initiative.mdx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+[February](/blog/bitcoin-grants-feb-2024), respectively.
+
+## Join Us in Our Mission
+
+We invite educators, developers, and advocates to [apply for
+grants](/apply/grant) under the OpenSats Education Initiative. Together, we can
+create a more informed and empowered society. For more information and to apply,
+please visit [opensats.org/apply](http://opensats.org/apply).
+
+Thank you for your continued support, and we look forward to seeing the
```
Comment (Zero-1729) [inline data/blog/announcing-the-opensats-education-initiative.mdx:114]: Minor nit

```suggestion
Thank you for your continued support. We look forward to seeing the
```

**Comment by Zero-1729** [comment]
LGTM ACK 🚀

**Pull request merged**